site/docs/contribute/modules.md

```diff
@@ -40,7 +40,13 @@ For any module chosen by the control plane to be part of the data path, the cont
 
 The names of the Kubernetes resources deployed by the module helm chart must contain the release name to avoid resource conflicts. A Kubernetes `service` resource which is used to access the module must have a name equal to the release name (this service name is also used in the optional [`spec.capabilities.api.endpoint.hostname`](../reference/crds.md#fybrikmodulespeccapabilitiesapiendpoint) field).
 
-Because the chart is installed by the control plane, the input `values` to the chart must match the relevant type of [arguments](../reference/crds.md#blueprintspecflowstepsindexarguments). 
+Because the chart is installed by the control plane, the input `values` to the chart will contain the following information:
+
+- `.Values.assets` - a list of [asset arguments](../reference/crds.md#blueprintspecmoduleskeyargumentsassetsindex) such as datastores, transformations, etc.
+- `.Values.selector` - [application selector](../reference/crds.md#blueprintspecapplication)
+- `.Values.context` - [application context](../reference/crds.md#blueprintspecapplication)
+- `.Values.labels` - labels specified in `FybrikApplication`
+- `.Values.uuid` - a unique id of `FybrikApplication` 
```

Comment:
maybe to add a link to the [values.sample.yaml](https://github.com/fybrik/arrow-flight-module/blob/cd168bb6cdf666c2ec1df960395c0dc1c8feeaa9/helm/afm/values.sample.yaml). I know that below there is a link to the entire chart, but the chart contains two values files and it can confuse.